ansible/roles/stack-sunbird/templates/content-service_application.conf

```diff
@@ -383,6 +383,9 @@ content {
     origin_data: ["name", "author", "license", "organisation"]
     props_to_remove: ["downloadUrl", "artifactUrl", "variants", "createdOn", "collections", "children", "lastUpdatedOn", "SYS_INTERNAL_LAST_UPDATED_ON", "versionKey", "s3Key", "status", "pkgVersion", "toc_url", "mimeTypesCount", "contentTypesCount", "leafNodesCount", "childNodes", "prevState", "lastPublishedOn", "flagReasons", "compatibilityLevel", "size", "publishChecklist", "publishComment", "LastPublishedBy", "rejectReasons", "rejectComment", "gradeLevel", "subject", "medium", "board", "topic", "purpose", "subtopic", "contentCredits", "owner", "collaborators", "creators", "contributors", "badgeAssertions", "dialcodes", "concepts", "keywords", "reservedDialcodes", "dialcodeRequired", "leafNodes", "sYS_INTERNAL_LAST_UPDATED_ON", "prevStatus", "lastPublishedBy", "streamingUrl"]
   }
+  media {
+    base_url: "{{content_media_base_url}}"
```

Comment:
you can have something like 
```
     base_url: "{{content_media_base_url | default(proto + '://' + domain_name)}}"
```
I think most probably in all cases this is gonna be the value for this variable. so we don't have to add one more entry to all.yaml

Reply:
changes done